Kitodo/src/main/java/org/goobi/production/chart/ProjectStatusDraw.java

```diff
@@ -14,11 +14,7 @@
 import de.intranda.commons.chart.results.DataRow;
 import de.intranda.commons.chart.results.DataTable;
 
-import java.awt.BasicStroke;
-import java.awt.Color;
-import java.awt.FontMetrics;
-import java.awt.GradientPaint;
-import java.awt.Graphics2D;
+import java.awt.*;
 import java.awt.geom.Line2D;
```

Comment:
What is the correct behaviour for existing checkstyle? How many similar import statements are needed before collapsing it with an star / asterix?